backend/api/src/DB/exams.go

```diff
@@ -145,3 +145,41 @@ func RemoveUserFromExam(db *gorm.DB, examID uint, userID uint) error {
 	}
 	return nil
 }
+
+// UnregisterFromExam function
+// Removes a user from an exams list of users.
+//
+// Or an error.
+func UnregisterFromExam(db *gorm.DB, examID uint, userID uint) error {
+	// find exam
+	var exam Exam
+	result := db.Where("id = ?", examID).First(&exam)
+	if result.Error != nil {
+		return result.Error
+	}
+	// find user
+	var user User
+	err := db.Where("id = ?", userID).First(&user).Error
+	if err != nil {
+		return err
+	}
+	// remove exam from user
+	err = db.Model(&user).Association("Exams").Delete([]Exam{exam})
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+// ListUserExams function
+// Lists the users exams which they have registered to
+//
+// Or an error
+func ListUserExams(db *gorm.DB, userID uint) ([]*Exam, error) {
```

Comment:
Verkar inte som den först hittar usern som den ska hitta exams för. Utan den hittar alla users exams tror jag. Måste göra en `db.Where("id = ?",userID).First(&user)` typ något sådant är kanske inte helt rätt, bara of the top of my head

Reply:
looks good now.